packages/docregistry/src/mimedocument.ts

```diff
@@ -133,7 +142,11 @@ export class MimeContent extends Widget {
     } else {
       data[this.mimeType] = model.toJSON();
     }
-    let mimeModel = new MimeModel({ data, callback: this._changeCallback });
+    let mimeModel = new MimeModel({
+      data,
+      callback: this._changeCallback,
+      metadata: { fragment: this._fragment }
```

Comment:
I think it would be useful to note in the documentation for `MimeModel` that `fragment` may be set in the metadata, and that renderers may choose to respect that (or not).

Reply:
It's useful indeed.